test/cabal-simple-debug/default.nix

```diff
@@ -20,7 +20,7 @@ let
 in recurseIntoAttrs {
   # DWARF only works on linux with GHC 8.10.2 and newer
   # GHC 9.2.1 disabled because of https://github.com/input-output-hk/haskell.nix/issues/1332
-  meta.disabled = __elem compiler-nix-name ["ghc865" "ghc884" "ghc921" "ghc922" "ghc923" "ghc924" "ghc925" "ghc926" "ghc927" "ghc941" "ghc942" "ghc943" "ghc944"]
+  meta.disabled = __elem compiler-nix-name ["ghc865" "ghc884" "ghc921" "ghc922" "ghc923" "ghc924" "ghc925" "ghc926" "ghc927"]
```

Comment:
so we are going to add back 941, 942, 943, 944?

Reply:
Only the compilers in ci.nix are actually tests (so this change will only add ghc944)